lib/exercism/team.rb

```diff
@@ -88,6 +88,13 @@ def dismiss(username)
     members.delete(user)
   end
 
+  def dismiss_invitation(username)
+    user = User.find_by_username(username)
```

Comment:
@tejasbubane Yes, I will finish the PR during the weekend. and I will let you know.